website/docs/cloud-docs/workspaces/create.mdx

```diff
@@ -0,0 +1,145 @@
+---
+page_title: Create workspaces - Workspaces - HCP Terraform
+description: >-
+  Workspaces organize infrastructure into meaningful groups. Learn how to create
+  and configure workspaces through the UI.
+---
+
+# Create workspaces
+
+This topic describes how to create and manage workspaces in HCP Terraform and Terraform Enterprise UI. A workspace is a group of infrastructure resources managed by Terraform. Refer to [Workspaces overview](/terraform/cloud-docs/workspaces) for additional information.
+
+> **Hands-on:** Try the [Get Started - HCP Terraform](/terraform/tutorials/cloud-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorials.
+
+## Introduction
+
+Create new workspaces when you need to manage a new collection of infrastructure resources.  You can use the following methods to create workspaces:
+
+- HCP Terraform UI: Refer to [Create a workspace](#create-a-workspace) for instructions.
+- Workspaces API: Send a `POST`call to the `/organizations/:organization_name/workspaces` endpoint to create a workspace. Refer to the [API documentation](/terraform/cloud-docs/api-docs/workspaces#create-a-workspace) for instructions.
+- Terraform Enterprise provider: Install the `tfe` provider and add the `tfe_workspace` resource to your configuration. Refer to the [`tfe` provider documentation](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/workspace) in the Terraform registry for instructions.
+- No-code provisioning: Use a no-code module from the registry to create a new workspace and deploy the module's resources. Refer to [Provisioning No-Code Infrastructure](/terraform/cloud-docs/no-code-provisioning/provisioning) for instructions.
+
+Each workspace belongs to a project. Refer to [Manage projects](/terraform/cloud-docs/projects/manage) for additional information.
+
+
+## Requirements
+
+You must be a member of a team with one of the following permissions enabled to create and manage workspaces:
+
+**Manage all projects**
+**Manage all workspaces**
+**Admin** permission group for a project.
```

Comment:
this is what it looks like right now

```suggestion
- **Manage all projects**
- **Manage all workspaces**
- **Admin** permission group for a project.
```

<img width="754" alt="image" src="https://github.com/user-attachments/assets/33ecf37d-e045-4ff0-9d1b-b384f00df4a0">